controllers/main.js

```diff
@@ -105,6 +108,7 @@ module.exports.sendPlayData = async (req, res, next) => {
     difficulty: req.body.difficulty,
     length: req.body.sessionLength,
     sessionData: [],
+    sessionEnd: false,
```

Comment:
Shouldn't this be `sessionEnded`?

Reply:
Yep, made a typo here, thanks for pointing it out